DataFormats/HcalRecHit/src/classes_def.xml

```diff
@@ -11,6 +11,12 @@
     <version ClassVersion="11" checksum="1379911758"/>
     <version ClassVersion="10" checksum="1697427050"/>
    </class>
+   <class name="HFQIE10Info" ClassVersion="3">
```

Comment:
This is not what the CMSSW docs say. See

https://twiki.cern.ch/twiki/bin/view/CMSPublic/SWGuideCreatingNewProducts#Class_versioning

On 05/05/2016 03:48 AM, Slava Krutelyov wrote:
>
> In DataFormats/HcalRecHit/src/classes_def.xml <https://github.com/cms-sw/cmssw/pull/14383#discussion_r62162258>:
>
> > @@ -11,6 +11,12 @@
> >      <version ClassVersion="11" checksum="1379911758"/>
> >      <version ClassVersion="10" checksum="1697427050"/>
> >     </class>
> > +   <class name="HFQIE10Info" ClassVersion="3">
>
> we recommend to start from 2.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub <https://github.com/cms-sw/cmssw/pull/14383/files/bb7a664cc1f3adac3512d57284b42aa643511ce5#r62162258>
>